apps/storefront/vercel.json

```diff
@@ -0,0 +1,39 @@
+{
```

Comment:
Does it work fine with the async rewrites() { we have in next.config.js?
Can we use one of them and not both?

Reply:
I think we can use both, as `https://dev-demo.saleor.io/checkout/` works fine. But maybe we should actually just use the Next.js redirects. I'll give it a try :)

Reply:
It works fine 👍